### PR TITLE
HPCC-13311 Add missing OPTIONs to docs

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@ghalliday please review

### CODE REVIEW

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 1]
```diff
@@ -1163,6 +1211,56 @@
...
+
+              <entry>Default: false</entry>
+
+              <entry>Retain the generated c++ files</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>spanMultipleCpp</emphasis></entry>
+
+              <entry>Default: false</entry>
```
Comment (ghalliday) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: default is true

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 2]
```diff
@@ -1163,6 +1211,56 @@
...
+
+              <entry>Generate a work unit in multiple c++ files </entry>
+            </row>
+
+            <row>
+              <entry><emphasis>activitiesPerCpp</emphasis></entry>
+
+              <entry>Default 1</entry>
```
Comment (ghalliday) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: Default is 500 for linux and 800 for windows.  I'm not sure how much detail is worth recording.

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 3]
```diff
@@ -1163,6 +1211,56 @@
...
+              <entry>Default: false</entry>
+
+              <entry>Generate information about inter-definition
+              dependencies</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>maxCompileThreads</emphasis></entry>
+
+              <entry>Default 2</entry>
```
Comment (ghalliday) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: Default is 1 for eclcc and 4 for eclccserver as far as I can see.

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 4]
```diff
@@ -643,6 +659,38 @@
               <entry>If true, look for common sub-expressions in
               TRANSFORMs/filters.</entry>
             </row>
+
+            <row>
+              <entry><emphasis>noteRecordSizeInGraph</emphasis></entry>
+
+              <entry>Default: false</entry>
```
Comment (ghalliday) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: default is true.

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 5]
```diff
@@ -643,6 +659,38 @@
...
+
+              <entry>Add distribution/sort orders to the graph</entry>
+            </row>
+
+            <row>
+              <entry><emphasis> showRecordCountInGraph</emphasis></entry>
+
+              <entry>Default: false</entry>
```
Comment (ghalliday) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: default true

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 6]
```diff
@@ -643,6 +659,38 @@
...
+            <row>
+              <entry><emphasis>showMetaInGraph</emphasis></entry>
+
+              <entry>Default: false</entry>
+
+              <entry>Add distribution/sort orders to the graph</entry>
+            </row>
+
+            <row>
+              <entry><emphasis> showRecordCountInGraph</emphasis></entry>
```
Comment (ghalliday) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: spurious space

**Comment by ghalliday** [comment]
@JamesDeFabia text is ok, but many of the defaults aren't correct.

**Comment by JamesDeFabia** [comment]
@ghalliday Edits made. Please have another look

**Pull request merged**